src/views/SendToken/SendToken.tsx

```diff
@@ -289,37 +299,47 @@ export function SendToken({ identity, onSuccess }: Props): JSX.Element {
     })();
   }, [amountBN, identity, recipient, tipBN]);
 
-  const handleAmountInput = useCallback((event) => {
-    const { value } = event.target;
+  const handleAmountInput = useCallback((event: FormEvent) => {
```

Comment:
Should it be more specific? I guess this will make the type casting below unnecessary.
```suggestion
  const handleAmountInput = useCallback((event: ChangeEvent<HTMLInputElement>)) => {
```

Reply:
And within the `if` condition we don’t need casting either